mteb/benchmarks/benchmarks.py

```diff
@@ -1432,3 +1432,224 @@ def load_results(
+            "AROVisualRelation",
+            "SugarCrepe",
+            "Winoground",
+            # VisualSTS
+            "STS12VisualSTS",
+            "STS13VisualSTS",
+            "STS14VisualSTS",
+            "STS15VisualSTS",
+            "STS16VisualSTS",
+            "VisualSTS17-Eng",
+            "VisualSTS17-Multilingual",
+            "VisualSTS-b-Eng",
+            "VisualSTS-b-Multilingual",
+            # Any2AnyRetrieval
+            "BLINKIT2IMultiChoice",
+            "BLINKIT2IRetrieval",
+            "BLINKIT2TMultiChoice",
+            "BLINKIT2TRetrieval",
+            "CIRRIT2IRetrieval",
+            "CUB200I2IRetrieval",
+            "EDIST2ITRetrieval",
+            "Fashion200kI2TRetrieval",
+            "Fashion200kT2IRetrieval",
+            "FashionIQIT2IRetrieval",
+            "Flickr30kI2TRetrieval",
+            "Flickr30kT2IRetrieval",
+            "FORBI2IRetrieval",
+            "GLDv2I2IRetrieval",
+            "GLDv2I2TRetrieval",
+            "HatefulMemesI2TRetrieval",
+            "HatefulMemesT2IRetrieval",
+            "ImageCoDeT2IRetrieval",
+            "ImageCoDeT2IMultiChoice",
+            "InfoSeekIT2ITRetrieval",
+            "InfoSeekIT2TRetrieval",
+            "MemotionI2TRetrieval",
+            "MemotionT2IRetrieval",
+            "METI2IRetrieval",
+            "MSCOCOI2TRetrieval",
+            "MSCOCOT2IRetrieval",
+            "NIGHTSI2IRetrieval",
+            "OVENIT2ITRetrieval",
+            "OVENIT2TRetrieval",
+            "ROxfordEasyI2IMultiChoice",
+            "ROxfordMediumI2IMultiChoice",
+            "ROxfordHardI2IMultiChoice",
+            "RP2kI2IRetrieval",
+            "RParisEasyI2IMultiChoice",
+            "RParisMediumI2IMultiChoice",
+            "RParisHardI2IMultiChoice",
+            "SciMMIRI2TRetrieval",
+            "SciMMIRT2IRetrieval",
+            "SketchyI2IRetrieval",
+            "SOPI2IRetrieval",
+            "StanfordCarsI2IRetrieval",
+            "TUBerlinT2IRetrieval",
+            "VidoreArxivQARetrieval",
+            "VidoreDocVQARetrieval",
+            "VidoreInfoVQARetrieval",
+            "VidoreTabfquadRetrieval",
+            "VidoreTatdqaRetrieval",
+            "VidoreShiftProjectRetrieval",
+            "VidoreSyntheticDocQAAIRetrieval",
+            "VidoreSyntheticDocQAEnergyRetrieval",
+            "VidoreSyntheticDocQAGovernmentReportsRetrieval",
+            "VidoreSyntheticDocQAHealthcareIndustryRetrieval",
+            "VisualNewsI2TRetrieval",
+            "VisualNewsT2IRetrieval",
+            "VizWizIT2TRetrieval",
+            "VQA2IT2TRetrieval",
+            "WebQAT2ITRetrieval",
+            "WebQAT2TRetrieval",
+            "WITT2IRetrieval",
+            "XFlickr30kCoT2IRetrieval",
+            "XM3600T2IRetrieval",
+        ],
+    ),
+    description="MIEB is a comprehensive image embeddings benchmark, spanning 8 task categories, covering 129 tasks and a total of 38 languages.",
```

Comment:
Description are displayed on the leaderboard so it might be nice to expand it a bit. Describing the type of tasks etc.

Reply:
Expanded.